conf/supervisord.conf.jnj

```diff
@@ -27,15 +27,23 @@ events = PROCESS_LOG
 [program:blobuploadcleanupworker]
 environment=
   PYTHONPATH=%(ENV_QUAYDIR)s
-command=python -m workers.blobuploadcleanupworker.blobuploadcleanupworker
+command={% if hotreload -%}
+  gunicorn --timeout=600 -b 'unix:/tmp/blobuploadcleanupworker.sock' -c %(ENV_QUAYCONF)s/gunicorn_worker.py 'workers.blobuploadcleanupworker.blobuploadcleanupworker:create_gunicorn_worker()'
```

Comment:
It doesn't really make sense to use gunicorn in these workers. These workers are not serving requests, and will end up timing out listening.

Reply:
gunicorn provides the hot reload functionality. Some Quay engineers use it in their development env, and I'd say we want to keep it.